code-of-conduct-languages/hi.md

```diff
@@ -0,0 +1,30 @@
+## CNCF सामुदायिक आचार संहिता v1.0
+
+### आचार संहिता का योगदानकर्ता
+
+इस परियोजना के योगदानकर्ताओं और अनुरक्षकों के रूप में, और एक खुले और स्वागत करने वाले समुदाय को बढ़ावा देने के हित में, हम उन सभी लोगों का सम्मान करने की प्रतिज्ञा करते हैं जो रिपोर्टिंग issues के माध्यम से योगदान करते हैं, फीचर अनुरोध पोस्ट करते हैं, प्रलेखन अपडेट करते हैं, pull requset या patches, और अन्य गतिविधियों में भाग लेते हैं।
+
+हम इस परियोजना में सभी के लिए उत्पीड़न-मुक्त अनुभव बनाने के लिए प्रतिबद्ध हैं, भले ही अनुभव के स्तर, लिंग, लिंग पहचान और अभिव्यक्ति, यौन अभिविन्यास, विकलांगता, व्यक्तिगत उपस्थिति, शरीर का आकार, नस्ल, जातीयता, आयु, धर्म, या राष्ट्रीयता।
```

Comment:
Also here, the meaning seems a bit distorted at last, you can change it to this:
....लिए प्रतिबद्ध हैं, बिना अनुभव के स्तर, लिंग, लिंग पहचान और अभिव्यक्ति, यौन अभिविन्यास, विकलांगता, व्यक्तिगत उपस्थिति, शरीर का आकार, नस्ल, जातीयता, आयु, धर्म, या राष्ट्रीयता को ध्यान में लाए।